### ISSUE TITLE
How to disable clickevent actions of the next month starting dates in current month

### ISSUE THREAD

**Issue by chinmay4github1987** [acro5piano/react-native-big-calendar#733]
Need to disable event press actions on the next month's dates in the current month.

**Comment by acro5piano** [comment]
Currently not possible. Feel free to open a pull request.

**Comment by charithAmila** [comment]
Hey @acro5piano 
What do you think about this feature? I will take a look if it feels worth it.

**Comment by acro5piano** [comment]
@charithAmila I think you can hide next month's events by adding a filtering logic. However, if it does not satisfy your needs, please go ahead! Thanks

**Comment by charithAmila** [comment]
We can add a prop for disabling event-clicking so users can disable their events as they need.

```
interface ICalendarEventBase {
    start: Date;
    end: Date;
    title: string;
    children?: ReactElement | null;
    hideHours?: boolean;
    disabled?: boolean; // <-- disabled prop
}
```

**Comment by acro5piano** [comment]
That sounds good!

**Comment by charithAmila** [comment]
@acro5piano PR is ready for review. https://github.com/acro5piano/react-native-big-calendar/pull/938

**Comment by acro5piano** [comment]
@charithAmila Thanks! Released on v4.5.0
